src/components/common/CardContainer/styles.js

```diff
@@ -40,52 +40,22 @@ const cityCardContainer = (display, theme) => {
     ? //mobile css
       {
         ...css,
-        display: 'flex',
-        flexDirection: 'row',
-        justifyContent: 'space-between',
-        flexWrap: 'nowrap',
+        // comented out for presentation
```

Comment:
why is this commented out?

Reply:
we were going to add some mobile style changes but ran out of time and had to present. 
We just kept kept a uniform desktop style at all breakpoints for the presentation